modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -5,16 +5,17 @@
 [id="metering-store-data-in-shared-volumes_{context}"]
 = Storing data in shared volumes
 
+Metering has no storage by default, but it can use any ReadWriteMany PersistentVolume or any StorageClass that provisions a ReadWriteMany PersistentVolume. A Network File System (NFS) implementation with distributed and parallel capability, such as Parallel NFS (pNFS), is supported if the NFS provider also supports the {product-title} implementation. Network latency issues might occur when using a pNFS third party provider for metering.
+
 [NOTE]
 ====
-NFS is not recommended to use with metering. 
+NFS and pNFS can fail to meet metering requirements and to provide the performance that is needed for the operator to work appropriately. Therefore, NFS and pNFS are not recommended to use in production.
```

Comment:
@chargio Thank you for your feedback. I updated the [NOTE] regarding NFS and pNFS not being recommended for production.